app/views/posts/index.html.erb

```diff
@@ -1 +1,42 @@
-<h1> Check this blogifyers posts</h1>
\ No newline at end of file
+<main>
+  <div class="card">
+    <h1><%= @user.name %></h1>
+    <h3>Number of posts: <%= @user.posts_counter %></h3>
+  </div>
+  <div>
+    <ul>
+      <% @posts.each do |post| %>
+        <%= link_to user_post_path(user_id: post.author_id, id: post.id) do %>
+        <div class="postContainer">
+          <li class="postCont">
+            <div class="post"> 
+              <h2 class="postTitle"><%= post.title %></h2>
+              <h4 class="postText"><%= post.text %></h4>
+              <div class="postCounters">
+                <p class="countText"> Comments: <%= post.comments_counter %> - Likes: <%= post.likes_counter %></p>
+              </div>
+            </div>
+          </li>
+          <div class="postCont">
+              <h3>Comments:</h3>
+              <ul>
+                <% recent_comments = post.comments.last(5) %>
+                <% if recent_comments.any? %>
+                  <% recent_comments.each do |comment| %>
```

Comment:
- [ ] As the previous reviewer highlighted, you have a method called `recent_comments` in your Post model. That is the method you should use. Also, we should avoid adding queries of this manner to our views.

  Kindly remove the assignment on line 23. You can use this to get the recent comments.

  ```
  if post.recent_comments.any?
    ...
  ```